src/labels/CustomLabels.labels

```diff
@@ -8727,6 +8735,30 @@ To check your existing settings, go to Duplicate Rules under Setup. For any rule
         <shortDescription>stgHelpUseDatedConvRates</shortDescription>
         <value>When checked, Dated Exchange Rates are used for currency conversion. If unchecked, the standard exchange rate is used even if Advanced Currency Management is enabled.</value>
     </labels>
+    <labels>
+        <fullName>stgInstallmentOppDisableAll</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
```

Comment:
@andrewyu-salesforce , I have just noticed this, please set new custom labels as protected and use `System.Label` when referencing them in Apex code.
@Julian88Tex , the PR is not ready for merge yet.

Reply:
@RadSFDeveloper thanks for the update. @andrewyu-salesforce I've set this ticket back to In Progress so it's current state is clear.